test/chart/BarChart.spec.tsx

```diff
@@ -323,4 +323,35 @@ describe('<BarChart />', () => {
       expect(rectangleProps).toHaveAttribute('height', '10');
     });
   });
+
+  test('Renders a (Bar) chart with less width than left, right margin and less height than top, bottom margin', () => {
```

Comment:
I had to read the description twice to understand. I think it would be helpful to add link to the github issue in the test name. Or a comment.

Reply:
ack, not the best name ever :D

Reply:
added comment